kpm/lib/kpm/kaui_artifact.rb

```diff
@@ -11,7 +11,7 @@ def versions(overrides={}, ssl_verify=true)
         coordinates = KPM::Coordinates.build_coordinates(coordinate_map)
         response    = REXML::Document.new nexus_remote(overrides, ssl_verify).search_for_artifacts(coordinates)
         versions    = SortedSet.new
-        response.elements.each('search-results/data/artifact/version') { |element| versions << element.text }
+        response.elements.each('searchNGResponse/data/artifact/version') { |element| versions << element.text }
```

Comment:
Since the endpoint was changed to "/lucene/search" the result search must be changed because the root element of the document returned is "searchNGResponse".  The "search-results" is for the endpoint "/data_index".